lighthergm/R/gof_lighthergm.R

```diff
@@ -273,7 +286,7 @@ gof_lighthergm <- function(net,
           seed_edgelist = network::as.edgelist(base_network),
           coef_within_block = coef_within_block,
           output = 'network',
-          ergm_control = ergm_control,
+          ergm_control = ergm_control_sim,
```

Comment:
`ergm_control_sim` here might be redundant?
https://github.com/sansan-inc/lighthergm/blob/d210cb1e84ad5213965e7f0fae9805923a16f4f7/lighthergm/R/gof_lighthergm.R#L253-L254

Could we simply write like below? Please let me know if I'm wrong.
```diff
-ergm_control_sim <- ergm_control 
-ergm_control_sim$MCMC.burnin <- interval 
+ergm_control$MCMC.burnin <- interval 
```

Reply:
@komatsuna4747 Let me just double check that it's not call by reference to avoid side-effects.